Resources/config/services.xml

```diff
@@ -19,9 +21,15 @@
     </parameters>
```

Comment:
For a 2.0 release, the class parameters could be removed. But let's do it in a different PR (later) to keep things focused here :)